fortuna/src/backend/routes/api/home.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
```

Comment:
Add //@flow strict at the top for when we use flow to check files.

See the server.js for an example